limitations.md

```diff
@@ -0,0 +1,13 @@
+# Limitations in iceoryx
+- limited Windows 10 support [\#33](https://github.com/eclipse-iceoryx/iceoryx/issues/33)
+- no support for 32-bit platforms, use at your own risk
```

Comment:
This can be remove as it's already in [installation.md](https://github.com/eclipse-iceoryx/iceoryx/blob/master/doc/website/getting-started/installation.md#dependencies).